### ISSUE TITLE
Compilation errors on Linux and vulkan-headers <1.2.197

### ISSUE THREAD

**Issue by Nindaleth** [Try/OpenGothic#252]
Environment: Fedora Linux 35, gcc 11.2.1, Mesa git (22.0.0-0.19.git44bdac9).

I'm getting Vulkan-related build errors such as the following ones. The previous time I've built was January 9 from b4ffb020 and that went successfully; I should be fully up to date including git submodules, also tried a clean rebuild with no change. There's about 500 lines of error output so it's entirely possible I'm missing something important, was a prerequisite added that's newly needed and I don't have it installed yet?

```
/home/black_fox/src/OpenGothic/lib/Tempest/Engine/gapi/vulkan/vcommandbuffer.cpp:264:23: error: ‘VK_STRUCTURE_TYPE_RENDERING_ATTACHMENT_INFO_KHR’ was not declared in this scope; did you mean ‘VK_STRUCTURE_TYPE_RENDER_PASS_ATTACHMENT_BEGIN_INFO_KHR’?
/home/black_fox/src/OpenGothic/lib/Tempest/Engine/gapi/vulkan/vcommandbuffer.cpp:238:39: error: ‘VK_STRUCTURE_TYPE_PIPELINE_RENDERING_CREATE_INFO_KHR’ was not declared in this scope; did you mean ‘VK_STRUCTURE_TYPE_PIPELINE_LIBRARY_CREATE_INFO_KHR’?
/home/black_fox/src/OpenGothic/lib/Tempest/Engine/gapi/vulkan/vcommandbuffer.cpp:228:5: error: ‘VkRenderingInfoKHR’ was not declared in this scope; did you mean ‘VkPipelineInfoKHR’?
/home/black_fox/src/OpenGothic/lib/Tempest/Engine/gapi/vulkan/vpipeline.cpp:92:44: error: ‘VkPipelineRenderingCreateInfoKHR’ does not name a type; did you mean ‘VkPipelineLibraryCreateInfoKHR’?
/home/black_fox/src/OpenGothic/lib/Tempest/Engine/gapi/vulkan/vdevice.h:241:5: error: ‘PFN_vkCmdEndRenderingKHR’ does not name a type; did you mean ‘PFN_vkCmdEndRenderPass2KHR’?
```

**Comment by Try** [comment]
Hi, @Nindaleth 

This `PFN_vkCmdEndRenderingKHR` and related stuff is one of the recently released vulkan extensions - make sure that you Vulkan-SDK is up to date.

**Comment by Nindaleth** [comment]
Ah, so it's related to the dynamic rendering extension? That would make sense as while the implementation in Mesa RADV is 1.3-compliant, the headers I have on my system are only 1.2.195 (dynamic rendering extension was released in 1.2.197).

I've updated to vulkan-headers-1.2.198 and that fixed the compilation. thanks! I can now compile and play as usual. I'll update the issue title to point to the actual root cause.

For any other Fedora users, the upstream `vulkan-headers` version in Fedora 35 is 1.2.189, the latest available in [che/mesa COPR repo](https://copr.fedorainfracloud.org/coprs/che/mesa/packages/) is 1.2.195 and there's also [koji](https://koji.fedoraproject.org/koji/packageinfo?packageID=27182) providing a 1.2.198 RPM for Fedora 36; it works just as well in Fedora 35 if you really need it now.
